shared/video-sdk/develop/live-streaming-over-multiple-channels/project-test/react-native.mdx

```diff
@@ -0,0 +1,67 @@
+<PlatformWrapper platform="react-native">
+
+### Test channel media Relay
+
+1.  In **App.tsx**, update `appID`, `channelName` and `destChannelName`.
+
+2. [Generate a temporary token](../reference/manage-agora-account#generate-a-temporary-token) in <Vpl k="CONSOLE" /> using `appID` and `channelName`. Use it to update `token` in `App.tsx`.
+
+3. Generate another token in <Vpl k="CONSOLE" /> using `appID` and `destChannelName`. Use it to update `destChannelToken` in `App.tsx`.
+
+4. In your browser, navigate to the <Link target="_blank" to="{{Global.DEMO_BASIC_VIDEO_CALL_URL}}"><Vg k="COMPANY" />web demo</Link> and update _App ID_, _Channel_, and _Token_ with the values for your temporary token, then click **Join**.
```

Comment:
Please add a space after the <Vg k="COMPANY" /> variable. It reads as Agoraweb demo..